tools/depends/target/ffmpeg/autobuild.sh

```diff
@@ -153,6 +153,8 @@ CFLAGS="$CFLAGS" CXXFLAGS="$CXXFLAGS" LDFLAGS="$LDFLAGS" \
 	--enable-libvorbis \
 	--enable-muxer=ogg \
 	--enable-encoder=libvorbis \
+        --enable-encoder=png \
```

Comment:
Jenkins was happy - but I will correct obviously. And bash scripts are tab / space unaware ...